src/__swaps__/utils/swaps.ts

```diff
@@ -383,21 +348,13 @@ export function niceIncrementFormatter({
     return valueBasedDecimalFormatter({
       nativePrice: inputAssetNativePrice,
       amount,
-      assetBalanceDisplay,
       roundingMode: 'up',
       precisionAdjustment: precisionBasedOffMagnitude(amount, isStablecoin),
       isStablecoin,
     });
   }
   if (percentageToSwap === 1) {
-    return valueBasedDecimalFormatter({
-      amount: inputAssetBalance,
-      nativePrice: inputAssetNativePrice,
-      roundingMode: 'up',
-      assetBalanceDisplay,
-      isStablecoin,
-      isMaxAmount: true,
-    });
+    return inputAssetBalance;
```

Comment:
this is where we are providing the `maxSwappableAmount` (which is passed in as the `inputAssetBalance`) when max is selected